patches/drizzle-orm@0.36.1.patch

```diff
@@ -88,6 +92,15 @@ index f1ce597e8a6e0e325788cac540a8fde32d36bb42..e6e2e6a65efa6f725d07264cc7c3a2a7
      const params = (0, import_sql.fillPlaceholders)(query.params, placeholderValues ?? {});
      logger.logQuery(query.sql, params);
      if (!fields && !customResultMapper) {
+@@ -127,7 +138,7 @@ class OPSQLitePreparedQuery extends import_session.SQLitePreparedQuery {
+   values(placeholderValues) {
+     const params = (0, import_sql.fillPlaceholders)(this.query.params, placeholderValues ?? {});
+     this.logger.logQuery(this.query.sql, params);
+-    return this.client.executeRawAsync(this.query.sql, params);
++    return this.client.executeRaw(this.query.sql, params);
```

Comment:
these need to be synchronous?

Reply:
Yes, the async methods don't actually exist